.dockerignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+**/node_modules
+package-lock.json
```

Comment:
package-lock.json nie powinien wchodzić do kontenera?

Reply:
Powinien. Teraz widzę, że w PR kasujemy też yarn.lock (przywrócę). 

Tworząc tę linię kierowałem się tym, że aktualnie nie ma w projekcie package-lock.json. W naszym zespole pracujemy z npmem ponieważ mieliśmy problemy z zależnościami instalowanymi przez yarna. 

TO DO:
- [ ] Przywrócić yarn.lock do projektu.
- [ ] Usunąć z .dockerignore package-lock.json.

Reply:
Done